app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
```

Comment:
Consider using `functools.lru_cache` for a more efficient and feature-rich caching mechanism. It provides additional features like limiting the cache size and clearing the cache.